projects/commudle-admin/src/app/feature-modules/users/components/public-profile/user-basic-details/email-preferences/email-preferences.component.ts

```diff
@@ -0,0 +1,74 @@
+import { ChangeDetectorRef, Component } from '@angular/core';
+import { EmailUnsubscribeGroupsService } from 'projects/commudle-admin/src/app/feature-modules/email-confirmations/services/email-unsubscribe-groups.service';
+import { AppUsersService } from 'projects/commudle-admin/src/app/services/app-users.service';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'app-email-preferences',
+  templateUrl: './email-preferences.component.html',
+  styleUrls: ['./email-preferences.component.scss'],
+})
+export class EmailPreferencesComponent {
+  subscriptionGroups: Object;
```

Comment:
Why have we not created a model for this?

Reply:
In future if we add more entities , it will automatically adjust those entities.